test/integration/minishift.go

```diff
@@ -95,16 +164,20 @@ func (m *Minishift) executingOcCommandSucceedsOrFails(command, expectedResult st
 	return nil
 }
 
-func (m *Minishift) executingMinishiftCommand(command string) error {
-	// TODO: there must be smarter way to destruct
-	cmdOut, cmdErr, cmdExit := m.runner.RunCommand(command)
-	lastCommandOutput = CommandOutput{
-		command,
-		cmdOut,
-		cmdErr,
-		cmdExit,
+func executingMinishiftCommandSucceedsOrFails(command, expectedResult string) error {
```

Comment:
will refactor to a general `SucceedOrFails(command function` by passing a func()